crates/cxx-qt-gen/src/generator/rust/qobject.rs

```diff
@@ -45,7 +47,10 @@ pub struct GeneratedRustQObject {
 }
 
 impl GeneratedRustQObject {
-    pub fn from(qobject: &ParsedQObject) -> Result<GeneratedRustQObject> {
+    pub fn from(
+        qobject: &ParsedQObject,
+        qualified_mappings: &BTreeMap<Ident, Path>,
+    ) -> Result<GeneratedRustQObject> {
```

Comment:
I think it is misleading to call this function `from` because it is not an impl of the core::convert::From trait. Can you come up with a new name for this function? Maybe `new`?

Reply:
Right, however note that this was copied from the others that are also `from` for both one and multiple parameters.

I can fix this as a separate commit at the end of this chain or a separate pull request...

Reply:
okay